### DISCUSSION TITLE
V1.1.2, I still don't care about cookies

### DISCUSSION THREAD

**Discussion by OhMyGuus** [OhMyGuus/I-Still-Dont-Care-About-Cookies#10512] [Announcements]
# I still don't care about cookies  V1.1.2

## (RECOMMENDED) Installation using official chrome & Firefox store
- [**Firefox** browser add-ons](https://addons.mozilla.org/en-US/firefox/addon/istilldontcareaboutcookies/
)
- [**Chrome** Web Store](https://chrome.google.com/webstore/detail/i-still-dont-care-about-c/edibdbjcniadpccecjdfdjjppcpchdlm)
- [**Edge** Add-ons](https://microsoftedge.microsoft.com/addons/detail/i-still-dont-care-about-/kkacdgacpkediooahopgcbdahlpipheh)

## Manual installation guide

- [Manual installation guide for Firefox](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/wiki/Firefox-installation-guide)
- [Manual installation guide for Chrome](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/wiki/Chrome-installation-guide)

## Thanks
Thanks a lot to the contributors who are willing to contribute to the project by either reporting websites or by willing to implement websites into the extension 👍 

## Contact
_If u need to contact me create a GH issue or message me on discord 
Discord: ThaGuus#2140_
Email: Info[AT]istilldontcareaboutcookies.com
Github: [Issues](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/issues/new)

## Special thanks to
- [appeasementPolitik](https://github.com/appeasementPolitik)
- [azgaresncf](https://github.com/azgaresncf)
- [winskil](https://github.com/winskil)
- [obfuscated-loop](https://github.com/obfuscated-loop)
- [CJYKK](https://github.com/CJYKK)
- [Erghel](https://github.com/Erghel)
And many more:
[Contributors](https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/graphs/contributors)
And ofcourse all the translators 💯 

## What's Changed
* [WIP] Reported websites - 16 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/2405
* [WIP] Reported websites - 17 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/2463
* Update css for idxrcookies (fixes #4518) by @fflorent in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/4520
* Fix k-ruoka.fi, k-rauta.fi etc rule (Kesko sites) by @ollisal-kesko in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/3748
* Update rules.js by @EmirLogas in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/3708
* Fixed Reported websites ~ 19 by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/2548
* [WIP] Fixed Reported websites ~ 18 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/2544
* Update rule for kick.com to support theater mode by @FMCorz in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/3296
* fixing multiple websites by @azgaresncf in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/9839
* Fix reported website - Add arduino.cc rule by @wwmike in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/7687
* Remove coop.co.uk rule that breaks site functionality and is redundant by @philwolstenholme in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/6409
* Add iconnectfx.com to rules.js by @obfuscated-loop in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/9386
* Update README.md by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10281
* Add rule for `store.duolingo.com` by @Voklen in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10301
* Update iconnectfx.com to be relative across pages by @obfuscated-loop in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10294
* fixing websites by @azgaresncf in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10412
* Fixed reported websites #19 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10402
* Prettier run by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10507
* Added android support by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10509
* Small fix (forgotten comma) by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10510
* Version update V1.1.2 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10511

## New Contributors
* @fflorent made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/4520
* @ollisal-kesko made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/3748
* @EmirLogas made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/3708
* @FMCorz made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/3296
* @azgaresncf made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/9839
* @wwmike made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/7687
* @philwolstenholme made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/6409
* @obfuscated-loop made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/9386
* @Voklen made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/10301

**Full Changelog**: https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/compare/v1.1.1...v1.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/releases/tag/v1.1.2'>V1.1.2, I still don't care about cookies</a>.</em>

**Comment by ghost** [comment]
it updates, yet it's still 1.1.1 version and not 1.1.2, people need to see that there is an update. please fix it as fast as possible ! thanks so much!
EDIT: talking about the *.xpi data. and so people do not recognise.

**Reply by OhMyGuus** [reply]
It's currently under review on all platforms, it should auto update as soon as they verify it.

**Reply by ghost** [reply]
Thanks you are back man, with a bigger team. I would help if i could. i can just post issues.
